features/polyExplorer/src/screens/stories/digitalGiantsStory.jsx

```diff
@@ -1,87 +1,57 @@
-import React from "react";
+import React, { useContext } from "react";
 
 import ClusterStory from "../../components/clusterStory/clusterStory.jsx";
-//import { ExplorerContext } from "../../context/explorer-context.jsx";
+import { ExplorerContext } from "../../context/explorer-context.jsx";
 import i18n from "../../i18n.js";
-import SectionTitle from "../../components/clusterStories/sectionTitle.jsx";
-import Tab from "../../components/clusterStories/tab.jsx";
+//import SectionTitle from "../../components/clusterStories/sectionTitle.jsx";
+//import Tab from "../../components/clusterStories/tab.jsx";
 
-import "./digitalGiantsStory.css";
-import BulletPoints from "../../components/bulletPoints/bulletPoints.jsx";
+const i18nHeader = "clusterDigitalGiantsStory";
 
-const i18nHeader = "digitalGiantsStory";
+const bigSixNames = [
```

Comment:
Is this the only place this is going to be used? Also, is this going to be a constant of is there some slim chance this might change with time?

Reply:
This is due to the actual company names being "Amazon Europe Sarl Corp (LU) Random Other Endless Titles"

Reply:
So yes for now this is the only place this is going to be used